sdk/identity/azure-identity/inc/azure/identity/client_secret_credential.hpp

```diff
@@ -25,7 +25,7 @@ namespace Azure { namespace Identity {
   /**
    * @brief Defines options for token authentication.
    */
-  struct ClientSecretCredentialOptions : public Azure::Core::_internal::ClientOptions
+  struct ClientSecretCredentialOptions : public Azure::Core::Credentials::TokenCredentialOptions
```

Comment:
Why was this change required instead of deriving directly from `ClientOptions`?

Reply:
It would look strange if there was CredentialOptions, and ClientSecretCredentialOptions wouldn't derive from it. Plus, other SDKs have it.

Reply:
Gotcha, makes sense.